exampleCourse/questions/fibonacciEditorV3/question.html

```diff
@@ -0,0 +1,32 @@
+<pl_question_panel>
+  <p>
+    The Fibonacci numbers are 1, 1, 2, 3, 5, 8, ..., where each
+    number is the sum of the two before it.
+    \[
+    F_n = F_{n-1} + F_{n-2}
+    \]
+  </p>
+  <p>
+    Write a Python function <tt>fib</tt> that takes a number <tt>n</tt>
+    and returns the n<sup>th</sup> Fibonacci number.
+  </p>
+</pl_question_panel>
+
+<script>
+window.FibonacciEditorConfig = function(editor) {
```

Comment:
I'd like to come up with some way to handle this without hard-coded global names. At some point we might want to put multiple questions on a page, and this will break.

For now, what if we just add another attribute to `pl_file_editor` like `ace_config="{fontSize: '10pt'}"`?

Reply:
So, I was really just trying to come up with a way to expose the `editor` object to courses so that they can use *anything* from the Ace API; setting config like this is just one example. I added mode and theme as attributes since those will be common, but I'd rather not have to duplicate the entire public-facing Ace API with attributes like that.

Reply:
Right, fair enough, I fully agree on the desire to give access to the ace editor object. I'm just nervous about hard-coded global window variable names. I'm sure we can come up with a good solution for this, but I want to avoid having courses start to use this particular pattern. Maybe just delete this from the example question for now?

Reply:
Sure, I'll get rid of it for now.